docs/operate/customize/compliance_info.md

```diff
@@ -40,28 +41,28 @@ You can also show information messages before and after the compliance checkboxe
 
 ```javascript
 import React from "react";
-import { i18next } from "@translations/invenio_rdm_records/i18next";
 import { SubmitReviewModal } from "@js/invenio_rdm_records";
 import { parametrize } from "react-overridable";
 
 const LegalDisclaimer = () => (
   <>
-    <p className="text-xs">
-      By publishing, you agree to our <a href="/terms" className="underline">terms of service</a>.
+    <p>
```

Comment:
I've removed the `className`s since they didn't do anything and distracted from the core of the example.